### ISSUE TITLE
"UnknownPropertyException" after clean installation

### ISSUE THREAD

**Issue by MartinSchmid2409** [kffein/Craft-export-Csv#19]
I just installed the plugin as trial on Craft CMS 3.4.30 (and Craft commerce 3.1.12). When I press the "Settings" button of the plugin (on page: "/admin/settings/plugins") I get this:

```
yii\base\UnknownPropertyException: Setting unknown property: kffein\craftexportcsv\controllers\SettingsController::request in I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\base\Component.php:209
Stack trace:
#0 I:\_dirsync\medisinn\medisinn_craft\vendor\kffein\craft-export-csv\src\controllers\SettingsController.php(44): yii\base\Component->__set('request', Object(craft\web\Request))
#1 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\base\BaseObject.php(109): kffein\craftexportcsv\controllers\SettingsController->init()
#2 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\base\Controller.php(86): yii\base\BaseObject->__construct(Array)
#3 [internal function]: yii\base\Controller->__construct('settings', Object(kffein\craftexportcsv\CraftExportCsv), Array)
#4 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\di\Container.php(392): ReflectionClass->newInstanceArgs(Array)
#5 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\di\Container.php(159): yii\di\Container->build('kffein\\craftexp...', Array, Array)
#6 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\BaseYii.php(345): yii\di\Container->get('kffein\\craftexp...', Array)
#7 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\base\Module.php(642): yii\BaseYii::createObject('kffein\\craftexp...', Array)
#8 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\base\Module.php(596): yii\base\Module->createControllerByID('settings')
#9 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\base\Module.php(588): yii\base\Module->createController('')
#10 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\base\Module.php(522): yii\base\Module->createController('settings')
#11 I:\_dirsync\medisinn\medisinn_craft\vendor\craftcms\cms\src\web\Application.php(291): yii\base\Module->runAction('craft-export-cs...', Array)
#12 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\web\Application.php(103): craft\web\Application->runAction('craft-export-cs...', Array)
#13 I:\_dirsync\medisinn\medisinn_craft\vendor\craftcms\cms\src\web\Application.php(276): yii\web\Application->handleRequest(Object(craft\web\Request))
#14 I:\_dirsync\medisinn\medisinn_craft\vendor\yiisoft\yii2\base\Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#15 I:\_dirsync\medisinn\medisinn_craft\web\de\index.php(21): yii\base\Application->run()
#16 {main}
```

Any help/fix appreciated.

**Comment by ed-kffein** [comment]
Hi @MartinSchmid2409, 
This issue is fixed in release 1.0.13, compatible with Craft 3.4.30
Thank you

**Comment by MartinSchmid2409** [comment]
👍
